template/web/src/utils/ipfs.ts

```diff
@@ -1,18 +1,128 @@
+import { cid } from 'is-ipfs';
```

Comment:
General thinking, I like a lot what's going on here.

Why some of these utility don't live in `is-ipfs` or, do we see other libraries have some of those?

Or should any of those live inside OnchainKit?

Reply:
@Yuripetusko thoughts on this?

Reply:
[`is-ipfs`](https://github.com/ipfs-shipyard/is-ipfs) is great, but it mostly just checks if url has ipfs, it doesn't have utils to replace gateway or add gateway and preserve the path etc. But regarding onchainkit I agree, I can create a similar PR there if this makes sense.

[I actually have all of this code in a library already](https://www.npmjs.com/package/@rmrk-team/ipfs-utils). Just need to add docs. Perhaps instead of having is-ipfs and all this code explicitly, our library will work? Otherwise I can just duplicate some of it in Onchainkit

Reply:
Let's probably move part of it inside OnchainKit.

We can proabably create a new section called IPFS.

Reply:
Ok will do tomorrow. I'll close this PR then and later will create a new one where onchainkit utils are used